pkg/controller/master/master_controller.go

```diff
@@ -341,7 +341,7 @@ func (s *jobSyncStrategy) GetJobFactory(owner metav1.Object) (controller.JobFact
 		return nil, err
 	}
 	return jobFactory(func(name string) (*v1batch.Job, *kapi.ConfigMap, error) {
-		vars, err := ansible.GenerateMachineSetVars(cluster, machineSet)
+		vars, err := ansible.GenerateClusterVars(cluster)
```

Comment:
Does using the machine set vars cause undesired behavior? Or does it simply include vars that are not needed?

Reply:
It causes undesired behavior, mainly because we're including whole structs in the vars.
For master install, the openshift_aws role is not used and some placeholders don't get resolved with actual values.

Reply:
OK. I think long term I would like something to be more resilient, whether it is our code or the playbooks. It doesn't make sense to me that when we are running a playbook against a machine set that we cannot use the machine set vars.

With that being said, I am fine with leaving it as you have it for now.

Reply:
We should be able to fix things once we get additional parameters in the aws role default vars. (At least for infra and master). For compute, we'll need to figure out what's the best way to structure the role vars so we don't have to include chunks of the config.